bindist/linux/debian/focal/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:focal
+
+ENV DEBIAN_FRONTEND=noninteractive LANG=C.UTF-8 LC_ALL=C.UTF-8
+
+RUN apt update \
+ && apt dist-upgrade -y \
```

Comment:
I'm still uneasy about the use of `dist-upgrade` in dockerfiles. The official Ubuntu docker repo keeps builds of different versions by date, e.g. `ubuntu:groovy-20201022.1` so I believe always pulling the newest `ubuntu:groovy` before building would give the most recent kernel updates anyway. If it is genuinely needed in our dockerfiles that's a bad sign IMO

Reply:
I'll remove it, but I swear it did solve an issue with GHDL before :(

Reply:
Possibly, but I think in that case it _should_ also be possible to pull the newer version of the base image and build (provided the only difference that matters is the kernel version)